gtsfm/loader/argoverse_dataset_loader.py

```diff
@@ -0,0 +1,132 @@
+""" Loader that reads images from Argoverse directories on disk.
+
+Authors: John Lambert
+"""
+
+import glob
+import os
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+from gtsam import Cal3Bundler, Point3, Pose3, Rot3
+
+from argoverse.data_loading.simple_track_dataloader import SimpleArgoverseTrackingDataLoader
+from argoverse.utils.calibration import get_calibration_config
+from argoverse.utils.se3 import SE3
+
+import gtsfm.utils.io as io_utils
+from gtsfm.common.image import Image
+from gtsfm.loader.loader_base import LoaderBase
+
+
+class ArgoverseDatasetLoader(LoaderBase):
+    """ """
+
+    def __init__(self, dataset_dir: str, log_id: str, stride: int = 10, max_num_imgs: int = 20) -> None:
+        """
+        Args:
+            dataset_dir: directory where raw Argoverse logs are stored on disk
+            log_id: unique ID of vehicle log
+            stride: sampling rate, e.g. every 2 images, every 4 images, etc.
+            max_num_imgs: number of images to load (starting from beginning of log sequence)
+        """
+        self.log_id = log_id
+        self.dl = SimpleArgoverseTrackingDataLoader(data_dir=dataset_dir, labels_dir=dataset_dir)
+
+        max_lookahead_sec = 60  # at original 30 fps frame rate, so 2 sec
```

Comment:
I do not get this. If its fps * time, then it should not be seconds, right?

Reply:
yeah let me rewrite that more clearly

Reply:
should be more clear now, let me know if not